### PR TITLE
Grid 2d find index performance

### PR DESCRIPTION
**Pull request by ecomodeller**

### CODE REVIEW

**Comment by ecomodeller** [comment]
`mikeio.Grid2D` only allow equidistant grid spacing, thus indices can be determined directly from the origin and grid spacing.

File: .github/workflows/perf_test.yml
Comment (jsmariegaard) [inline .github/workflows/perf_test.yml:1]: ❤️

**Review by jsmariegaard — APPROVED** [review]
🔥 🔥 🔥

**Comment by jsmariegaard** [comment]
:shipit:

**Pull request merged**